test/docker-worker-test

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env babel-node
+
+import program from 'commander';
+import { spawn } from 'child_process';
+import path from 'path';
+import diveSync from 'diveSync';
+
+let baseDir = path.resolve(__dirname, '..');
+let binary = `${baseDir}/node_modules/mocha/bin/mocha`;
```

Comment:
Note, if you added this script as `npm test` then `node_modules/.bin/` would be in `$PATH`. But I suspect the runner is different when running locally maybe...

Afaik it's still possible to pass arguments to `npm test`, it's just `npm test -- --this-chunk 5` but I could be remembering that wrong too...

Anyways, nit... this is not important at all...